### DISCUSSION TITLE
osu! reading storyboard folders as elements

### DISCUSSION THREAD

**Discussion by nolife99** [ppy/osu#16374] [Q&A]
in the verify tab in the editor, AiMod reads storyboard folders as the elements themselves, causing a 0-byte file error
![image](https://user-images.githubusercontent.com/89430659/148666603-daefc9dd-ccc4-422d-984b-fbb3d952621b.png)
playing the map with the storyboard on works though???
![image](https://user-images.githubusercontent.com/89430659/148666658-7cc83c54-c588-4e4f-8cf9-2aa2eff2b800.png)

the beatmap is http://osu.ppy.sh/s/1054045 and no storyboard elements have been modified by me

current release: 2022.108.0

Log files:
[performance.log](https://github.com/ppy/osu/files/7834364/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7834365/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7834366/updater.log)
[database.log](https://github.com/ppy/osu/files/7834367/database.log)
[network.log](https://github.com/ppy/osu/files/7834368/network.log)

**Comment by peppy** [comment]
This looks quite deep. The directories are actually being imported as files somehow.

**Comment by peppy** [comment]
@nolife99 I can't reproduce this. Can you try deleting the beatmap, restarting lazer, reimporting and then testing again?

**Reply by nolife99** [reply]
I'm using windows 10 and I tried redownloading it from the website and it still says the same thing, a 0-byte file instead of reading the folder. But the storyboard still works
I'm including the .osz in case
https://www.mediafire.com/file/yqgwtpm15a0d8in/1054045_a_hisa_-_Alexithymia_Lupinus_Tokei_no_Heya_to_Seishin_Sekai.osz/file

**Reply by peppy** [reply]
Are you saying you're not using the .osz from the official download link?

**Reply by nolife99** [reply]
No, I downloaded it from the osu website and just included the file in case
before I downloaded it from osu direct in game

**Reply by bdach** [reply]
@nolife99 could you try deleting the map, restarting the game and only *then* importing it back in? does that change anything?

fwiw i've attempted to reproduce on windows and linux and couldn't.

**Reply by nolife99** [reply]
No that doesn't change anything. 
It only seems to happen when you import it using the import files under debug in settings

**Reply by bdach** [reply]
i've tried using said button on multiple operating system and still am unable to reproduce.
